samples/bluetooth/throughput/src/main.c

```diff
@@ -364,7 +364,7 @@ static void test_run(void)
 	delta = k_uptime_delta_32(&stamp);
```

Comment:
Since delta switched type and this function is deperecated, we should probably switch to `k_uptime_delta`.

Reply:
done